pkg/image/definition.go

```diff
@@ -16,6 +16,7 @@ type Definition struct {
 	APIVersion      string          `yaml:"apiVersion"`
 	Image           Image           `yaml:"image"`
 	OperatingSystem OperatingSystem `yaml:"operatingSystem"`
+	Hauler          Hauler          `yaml:"hauler"`
```

Comment:
Hauler is the implementation - not the API contract we want to expose to the user. This should be `workloads` or something similar where it's clear what goes into it.

Reply:
Exactly this ^^